docs/integrations/rabbitmq.md

```diff
@@ -59,3 +61,40 @@ orka({
   }
 });
 ```
+
+## RequestContext
+When you use `BaseQueueHandler`, orka by default runs the message handling inside `runWithContext` function and appends the `correlationId` of the message in the context.
+
+That means that every log of your consumer will contain the `correlationId` of the message and that you can also add any variable you want in the context.
+
+Example:
+```js
+const { BaseQueueHandler } = require('rabbit-queue');
+const { getRabbit, getRequestContext } = require('@workablehr/orka');
+
+async function handler(m) {
+  logger.info('Var: ', getRequestContext().get('test-var'));
+}
+
+class ExampleHandler extends BaseQueueHandler {
+  constructor(queueName, logEnabled = true) {
+    const config = require('./config');
+    super(queueName, getRabbit(), {
+      logEnabled,
+      retries: 3, // 3 retries
+      retryDelay: 1000 // in ms
+    });
+  }
+
+  async handle(message) {
+    getRequestContext().set('test-var', 'orka');
+    logger.info(message);
+    await handler(message);
+  }
+
+}
+
+// In the above example if we receive a message with correlationId=xxx, and message="hello" it will log:
+// [xxx] hello
+// [xxx] Var: orka
```

Comment:
❓ Is it on two different lines? Not in the same line?

Reply:
They are 2 logs